string station_id
string station_status
Pose position